src/main/scala/org/broadinstitute/dsde/workbench/leonardo/dao/google/HttpGoogleDataprocDAO.scala

```diff
@@ -189,8 +190,10 @@ class HttpGoogleDataprocDAO(appName: String,
   private def getClusterConfig(machineConfig: MachineConfig, initScript: GcsPath, clusterServiceAccount: Option[WorkbenchEmail], credentialsFileName: Option[String], stagingBucket: GcsBucketName): DataprocClusterConfig = {
     // Create a GceClusterConfig, which has the common config settings for resources of Google Compute Engine cluster instances,
     // applicable to all instances in the cluster.
-    // Set the network tag, which is needed by the firewall rule that allows leo to talk to the cluster
-    val gceClusterConfig = new GceClusterConfig().setTags(List(defaultNetworkTag.value).asJava)
+    // Set the network tag, network, and subnet. This allows the creaed GCE instances to be exposed by Leo's firewall rule.
+    val gceClusterConfig = new GceClusterConfig()
+      .setTags(List(defaultNetworkTag.value).asJava)
+      .setSubnetworkUri(defaultVPCSubnet.value)
```

Comment:
Hey @eap, we observed this breaking on some projects where the "default" subnet doesn't exist. I'm thinking of making the `firewallVPCSubnet` config param optional, and _not_ calling `GceClusterConfig.setSubnetworkUri()` if it's not present. If I understand the docs correctly, this will trigger "auto" subnet creation as it was doing before. The config will still be available if you want to explicitly set it. Does this make sense?